test/common/vmimages.py

```diff
@@ -114,9 +118,74 @@ def enough_disk_space():
     free = st.f_bavail * st.f_frsize / (1024*1024*1024)
     return free >= PRUNE_THRESHOLD_G;
 
+def get_refs():
+    """Return dictionary for available refs of the format {'rhel-7.4': 'ad50328990e44c22501bd5e454746d4b5e561b7c'}
```

Comment:
For the processing I agree, but I find the debug output (in the line `Considering images from [...]`) more helpful with the name.